.github/workflows/e2e.yaml

```diff
@@ -22,4 +22,11 @@ jobs:
 
     - name: Run e2e tests
       run: |
-        make e2e
+        make -k e2e
```

Comment:
Mind commenting why `-k` is here? I liked your explanation in the comments and think it would be worth putting in here.